code/game/objects/structures/signs/signs_maps.dm

```diff
@@ -4,6 +4,12 @@
 	name = "station map"
 	desc = "A framed picture of the station."
 	max_integrity = 500
+	
+/obj/structure/sign/map/examine(mob/user)
+	..()
+	if(prob(1) || (SSevents.holidays && SSevents.holidays[APRIL_FOOLS])) // TODO: Enable at all times in hypercube-station
+		say("You are here.")
+		visible_message("points at [user].")
```

Comment:
Okay I'll relent on the other thing, but use the proper pointing procs

Reply:
We have those? Cool, will do

Reply:
That's a /living proc you meme @Cyberboss